### PR TITLE
[Feature #158761414] Deactivate event

### PR DESCRIPTION
**Pull request by OsayamenOsaretin**
#### What Does This PR Do?
Add feature for Deactivate event using GraphQL mutation.
#### Description Of Task To Be Completed
- Add test for `deactivateEvent` mutation and `eventsList` query after event is deactivated.
- Add mutation for `deactivateEvent` and filter event-list for active events only.

#### Any Background Context You Want To Provide?
- Deactivating events should be implemented as a `Delete Event` feature to non-admin end users on the front end. It is only implemented as Deactivate event here to portray to the backend that the event is not being deleted completely but only deactivated from the front-end view.

- I have chosen snapshot testing for [these reasons](http://docs.graphene-python.org/en/latest/testing/#snapshot-testing)

#### How can this be manually tested?
- run `tox`
- run the app, deactivate an event, and query for the event list, confirming that the event is no longer a Node returned by the `eventsList` query.

#### What are the related pivotal tracker stories
[#158761414](https://www.pivotaltracker.com/story/show/158761414)

### CODE REVIEW

**Comment by OsayamenOsaretin** [comment]
The tests seem to fail because of a dependency issue on CircleCI. I am guessing that the dependency caching doesn't somehow allow new dependencies to be installed, but I am not sure about this.

File: server/graphql_schemas/tests.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    def setUp(self):
+        self.event_creator = create_user("creatorId")
+        self.non_event_creator = create_user("ncreatorId")
+        self.admin = create_admin_user("adminId")
+        self.category = create_category()
+        create_event(self.event_creator, self.category)
+
+    def tearDown(self):
+        Event.objects.all().delete()
+        GoogleUser.objects.all().delete()
```
Comment (makindemayowa) [inline server/graphql_schemas/tests.py]: Category should be torn down too after the test.

**Comment by tosmak16** [comment]
Well done @OsayamenOsaretin but please let's have our test folder structure in graphql_schma app to match API app test folder structure.  ` “Flat is better than nested,” this makes it easier to navigate a Django app.`

File: server/graphql_schemas/tests.py (hunk 1 above)
Comment (OsayamenOsaretin) [reply server/graphql_schemas/tests.py]: @makindemayowa. It doesn't really save to the database, just creates an instance of it. I didn't think there would be any need to tear it down.

**Pull request merged**